public/js/buildGherkin.js

```diff
@@ -297,7 +301,7 @@ function buildTable(testEl) { // el should be the test element
     var scenario = $(testEl).children('.testSteps');
     scenario.children('.testStep').each(function() {
         $(this).children('input, select').each(function() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this function to use "return" consistently. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AS3801)